pyproject.toml

```diff
@@ -60,13 +106,54 @@ extend-ignore = [
     "A005",
     # TODO: Consider using copyright headers
     "CPY",
+    # Not Applicable
+    "DJ",
+    # It's fine to have TODOs
+    "FIX",
+    # Not Applicable
+    "INT",
+    # Even prevents pytest.raises around an iteration which is too strict
+    "PT012",
+    # Triggers for protocol implementations that don't need the arg too
+    "ARG002",
+    # Complexity checks usually reduce readability
+    "C90",
+    # Not Applicable
+    "NPY",
+    # Not Applicable
+    "PD",
+    # We use the Exception suffix instead
```

Comment:
Also this one seems to be that `Error` suffix is more conventional than `Exception` suffix. I think it's less of an issue though